Makefile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 The ObjectHash-Proto Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: test
+test:
+	go test -cover -covermode=count -coverprofile=coverage.out .
+	sed -i "s/_$${PWD//\//\\\/}/./" coverage.out
```

Comment:
Change GOPATH instead of sed magic?

Reply:
I used this when I was developing locally on my machine to make the relative imports work. It's no longer needed (See 0758489 for the fix)